Makefile.client

```diff
@@ -146,7 +146,8 @@ grpc: $(LIBGRPCDIR)/libgrpc_indicator
 
 $(LIBGRPCDIR)/libgrpc_indicator:
 	mkdir -p $(LIBGRPCDIR)
-	git clone -b $(shell curl -L https://grpc.io/release) \
+  # temporarily set to 1.19 due to build problems with 1.20
```

Comment:
This is going in as part of https://github.com/NVIDIA/tensorrt-inference-server/pull/240 so need to remove it from here before merging.